.pipelines/cni/load-test-templates/validate-state-template.yaml

```diff
@@ -12,9 +13,19 @@ steps:
       scriptType: "bash"
       addSpnToEnvironment: true
       inlineScript: |
-        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        if [ ${{ parameters.cnsManagedEndpoint }} == "true" ] && [ ${{ parameters.restartCase }} == "true" ]
+        then
+          echo If this step fails, test manually.
+          echo Only fails when CNS is managing the endpoint state and state file does not existing due to node restart with 0 pods, outside of kube-system pods, scheduled on node.

```

Comment:
maybe specify pods we're looking for instead of `outside of kube-system pods`

Reply:
I will put that we are looking for load-test pods in this test.